barretenberg/cpp/src/barretenberg/honk/proof_system/protogalaxy_prover.hpp

```diff
@@ -278,8 +285,32 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
             Utils::add_nested_tuples(univariate_accumulators, accumulators);
         }
         // Batch the univariate contributions from each sub-relation to obtain the round univariate
-        return Utils::template batch_over_relations<ExtendedUnivariateWithRandomization>(
-            univariate_accumulators, alpha, pow_univariate);
+        return Utils::template batch_over_relations<ExtendedUnivariateWithRandomization>(univariate_accumulators,
+                                                                                         alpha);
+    }
+
+    static Univariate<FF,
+                      (Flavor::BATCHED_RELATION_TOTAL_LENGTH - 1) * (ProverInstances::NUM - 1) + 1,
+                      ProverInstances::NUM>
+    compute_combiner_quotient(FF compressed_perturbator, ExtendedUnivariateWithRandomization combiner)
+    {
+        std::array<FF, (Flavor::BATCHED_RELATION_TOTAL_LENGTH - 2) * (ProverInstances::NUM - 1)>
+            combiner_quotient_evals = {};
+
+        // we need to evaluate at values not part in the vanishing set
+        for (size_t point = ProverInstances::NUM; point < combiner.size(); point++) {
+            auto idx = point - ProverInstances::NUM;
+            auto lagrange_0 = FF(1) - FF(point);
```

Comment:
You need to create a univariate from $F(\alpha)L_0(X)$ and barycentrically extend. Example: $1-x$ is the Lagrange polynomial centered at $0$ on the domain $\{0,1\}$, and the barycentric extension of $15L_0(X)$ to $\{0,1,2,3\}$ is $\{15,0,-15,-30\}$ but it looks to me here like you extend it constantly to $15$ on $2$ and $3$.

Reply:
15(1 - 2) = -15 and 15(1 - 3) = -30. Maybe I'm brain dead but I don't seen how this is wrong.

Reply:
Ah, you're right, didn't read carefully enough.

Reply:
✅